live-site/components/welcome-section/team-race/Team.styles.ts

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components';
+import { H3, P } from '../../../../shared-ui/style/typography';
+import { colors } from '../../../../shared-ui/style/colors';
+import { max } from '../../../../shared-ui/lib/responsive';
+
+const StyledTeamContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  @media ${max.tablet} {
+    flex-direction: column;
+  }
+`;
+
+const StyledTeamHeader = styled(H3)`
+  color: ${colors.WHITE};
+`;
+
+const StyledTeamName = styled(P)`
+  text-transform: uppercase;
+  color: ${colors.BUTTON_GREEN};
+  font-size: 1.5em;
```

Comment:
It's awesome that all of the car lines + team names are the same width now BUT we still have issues at breakpoints like this where either the team name or points split into two lines.

To fix this, let's just make sure we always keep them on the same line. So you can fix this by making the font smaller at certain breakpoints. But make sure that all the text is ALWAYS on one line so the styling doesn't get wacky
<img width="750" alt="Screenshot 2023-01-23 at
<img width="123" alt="Screenshot 2023-01-23 at 12 32 39 AM" src="https://user-images.githubusercontent.com/59738880/213971903-ec14cf18-ae26-4a96-a03b-4c391db1ae9d.png">
 12 31 49 AM" src="https://user-images.githubusercontent.com/59738880/213971825-58495b89-e6f6-4d7f-b2cf-4cab788a4aea.png">